docs/docs/reference/dropped/weak-conformance-spec.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: doc-page
+title: Dropped: Weak Conformance - More Details
+---
+
+To simplify the underlying type theory, Dotty drops the notion of weak
+conformance altogether. Instead, it provides more flexibility when
+assigning a type to a constant expression. The new rule is:
```

Comment:
I think it would be good if this rule declared its name, which I think is "harmonize" (and also known as "new weak conformance").